cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,442 @@
+If you would like to annotate a task with a custom model,
+please convert it to the intermediate representation (IR) format via the model optimizer tool.
+See [OpenVINO documentation](https://software.intel.com/en-us/articles/OpenVINO-InferEngine) for details.
+You can "register" a model and "use" it after that to pre annotate your tasks.
 
-    __Overlap Size__. Use this option to make overlapped segments. The option makes tracks continuous from one segment into another. Use it for interpolation mode. There are several use cases for the parameter:
-    - For an interpolation task (video sequence) if an object exists on overlapped segments it will be automatically merged into one track if overlap is greater than zero and annotation is good enough on adjacent segments. If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file you will have several tracks, one for each segment, which correspond to the object).
-    - For an annotation task (independent images) if an object exists on overlapped segments bounding boxes will be automatically merged into one if overlap is greater than zero and annotation is good enough on adjacent segments. If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file you will have several bounding boxes for the same object.
+![](static/documentation/images/image099.jpg)
 
-    Thus you annotate an object on the first segment. You annotate the same object on second segment and if you do it right you will have one track inside your annotation file. If annotations on different segments (on overlapped frames) are very different or overlap is zero you will have two tracks for the same object. This functionality works only for bounding boxes. Polygon, polyline, points don't support automatic merge on overlapped segments even the overlap parameter isn't zero and match between corresponding shapes on adjacent segments is perfect.
+The model manager allows you to manage your deep learning (DL) models uploaded for auto annotation.
+Using the functionality you can upload, update or delete a specific DL model.
+Use "Auto annotation" button to pre annotate a task using one of your DL models. [Read more](/apps/auto_annotation)
 
-    __Segment size__. Use this option to divide a huge dataset into a few smaller segments.
+![](static/documentation/images/image104.jpg)
 
-    __Image Quality__. Use this option to specify quality of uploaded images. The option makes it faster to load high-quality datasets. Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
+### Search
 
-    Push ``Submit`` button and it will be added into the list of annotation tasks. Finally you should see something similar to the figure below:
+There are several options how to use the search.
+It may be task's name or id, owner's name, task's status, assignee's name and so on. The search is case insensitive.
```

Comment:
".. and so on" doesn't sound good (only "mode" hasn't been mentioned). Additionally it would be better, provide some examples here.